src/test/java/com/hubspot/jinjava/tree/FailOnUnknownTokensTest.java

```diff
@@ -40,4 +35,25 @@ public void itReplaceTokensWithoutException() {
     String renderedTemplate = jinjava.render(template, context);
     assertThat(renderedTemplate).isEqualTo("hello test and test1");
   }
+
+  @Test
+  public void itReplacesTokensInContextButThrowsExceptionForOthers() {
```

Comment:
could you break this up into two separate tests?

Reply:
on it :+1: